src/main/resources/com/google/api/codegen/ruby/smoke_test.snip

```diff
@@ -23,7 +23,15 @@ describe "{@smokeTest.name}" do
       project_id = ENV["SMOKE_TEST_PROJECT"].freeze
     @end
 
-    {@sampleCodeCore(smokeTest.apiMethod)}
+    process_response = ->(response) {
```

Comment:
I still don't think we should put a dummy process_response definition here. This way we're forcing the caller of the samplegen to provide a response handler. Why can't you call 'process_xxx' after {@sampleCodeCall(methodView)} in test.snip just  for the unit test purpose?